app/assets/stylesheets/nav.scss

```diff
@@ -0,0 +1,9 @@
+.search-bar span {
+  display: inline-block;
+}
+
+.search-button {
+  background-color: white;
+  color: #088da5;
```

Comment:
Color literals like `#088da5` should only be used in variable declarations; they should be referred to via variable everywhere else.